### PR TITLE
Move PHPUnit Tests to GA

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Move PHPUnit Tests to GitHub Action. Also test on php5.6 and php7.2!
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? |
| How to test?  | GA should pass

### CODE REVIEW

**Comment by matks** [comment]
The GA pass on my fork https://github.com/matks/autoupgrade/pull/3

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: PHP tests
+on: [push, pull_request]
+jobs:
+  build:
+    strategy:
+      matrix:
+        php-versions: ['5.6', '7.2']
```
Comment (atomiix) [inline .github/workflows/php.yml:7]: Maybe add 7.3 as it should be compatible with PrestaShop 1.7.7.0 that is itself compatible with php 7.3?

Comment (matks) [reply .github/workflows/php.yml]: I'm going to even see if it works for php7.4. 1.7.8 must be php7.4 compatible.

**Review by atomiix — APPROVED** [review]

**Comment by matks** [comment]
Thanks for review!

**Pull request merged**